operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -508,7 +508,8 @@ alertmanager:
   # -- Pod affinity settings for the alertmanager. This is ignored when alertmanager.zoneAwareReplication.enabled=true.
   affinity: {}
 
-  # -- topologySpreadConstraints allows to customize the default topologySpreadConstraints. This can be either a single dict as shown below or a slice of topologySpreadConstraints.
+  # -- topologySpreadConstraints allows to customize the default topologySpreadConstraints. This can be either a single dict as shown below or a slice of topologySpreadConstraints. 
```

Comment:
This comment has a space appended to it. I think the linter is complaining about it. Same applies to all the other places that you added minDomains to